Bottom: 817bc7fc2d6c19c9f8ab26da2649cbb2a444059d
Top:    836ac9a3fedd953980f1b878e23d7b18de86140b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-01-24 19:09:53 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
